releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,33 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.2.0 - January 13, 2020
+
+* Simplified Feature Management config file format.
```

Comment:
What we give here is just an example. Is the formatting schema published somewhere?

Reply:
We don't have a published schema for the Feature Management configuration file. I will start working on one to add on with the Azure App Configuration one.